Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -4,16 +4,29 @@
 extension Style {
 
     /// This function creates an expression that will localize the `textField` property of a `SymbolLayer`
-    /// - Parameter locale: A `SupportedLanguage` based `Locale`
-    internal func localizeLabels(into locale: Locale) throws {
+    /// - Parameter locale: The`Locale` to update the map to
+    /// - Parameter layerIds: An optional list of ids that need to be localized. If nothing is provided, all layers will be updated
```

Comment:
```suggestion
    /// - Parameter layerIds: An optional list of ids that need to be localized. If `nil` is provided, all layers will be updated
```
This may be more clear so that folks are less likely to think they can pass `[]` to update all layers